src/lib/core/CHIPConfig.h

```diff
@@ -1785,6 +1785,20 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_MINMDNS_DYNAMIC_OPERATIONAL_RESPONDER_LIST 0
 #endif // CHIP_CONFIG_MINMDNS_DYNAMIC_OPERATIONAL_RESPONDER_LIST
 
+/**
+ *  @def CHIP_CONFIG_IM_STATUS_CODE_VERBOSE_FORMAT
+ *
+ *  If 1, IM status codes, when logged, will be formatted as "0xNN (NameOfCode)"
+ *  If 0, IM status codes, when logged, will be formatted as "0xNN" In either
+ *  case, the macro ChipLogFormatIMStatus expands to a suitable printf format
+ *  string, which already includes the '%' in it, to be used with
+ *  ChipLogValueIMStatus(status).
+ */
+
+#ifndef CHIP_CONFIG_IM_STATUS_CODE_VERBOSE_FORMAT
+#define CHIP_CONFIG_IM_STATUS_CODE_VERBOSE_FORMAT 1
```

Comment:
This should be 0 by default to save flash, and only be enabled on Darwin/Linux

Reply:
I was going back and forth on that....  I can do that.